app/prepare_proposal.go

```diff
@@ -15,15 +15,13 @@ import (
 )
 
 // PrepareProposal fulfills the celestia-core version of the ABCI interface by
-// preparing the proposal block data. The square size is determined by first
-// estimating it via the size of the passed block data. Then, this method
-// generates the data root for the proposal block and passes it back to
-// tendermint via the BlockData. Panics indicate a developer error and should
-// immediately halt the node for visibility and so they can be quickly resolved.
+// preparing the proposal block data. This method generates the data root for
+// the proposal block and passes it back to tendermint via the BlockData. Panics
+// indicate a developer error and should immediately halt the node for
+// visibility and so they can be quickly resolved.
```

Comment:
Removed a sentence about square size estimating b/c that doesn't seem like an explicit step in this function